airesources/CSharp/Halite2/hlt/Log.cs

```diff
@@ -17,7 +17,7 @@ public static void Initialize(TextWriter f)
             instance = new Log(f);
         }
 
-        public static void Log(string message)
+        public static void LogMessage(string message)
```

Comment:
I would prefer you keep it as "Log", LogMessage is a bit too verbose and we don't have any other types of logging that need to be distinguished.

Reply:
I would leave it as LogMessage; with Log I'm getting this error: *'Log': member names cannot be the same as their enclosing type*; I had to change Log to LogAction in my local code to make it work.

Reply:
Roger. Makes sense.